tests/evaluation/aggregates7/IndexAggregateMax.csv

```diff
@@ -0,0 +1 @@
+1	100
```

Comment:
It would be nice if you could tweak the datalog so that each aggregate produced a different value. Here count and max are the same.